src/MeshManager.cpp

```diff
@@ -451,6 +467,12 @@ namespace blitzdg {
 
     void MeshManager::readElements(const string& E2VFile) {
         EToV = csvread<index_type>(E2VFile, NumElements, NumFaces);
+
+        BCType = index_vec_smart_ptr(new index_vector_type(NumElements*NumFaces));
+        EToE = index_vec_smart_ptr(new index_vector_type(NumElements*NumFaces));
+        EToF = index_vec_smart_ptr(new index_vector_type(NumElements*NumFaces));
+        buildConnectivity();
+        buildBCTable(3);
     }
```

Comment:
Same as above, does 3 equal `NumFaces`?

Reply:
On line 475, 3 refers to a tag number. I've added the long-awaited enum to clear this up.